contracts/.env.example

```
@@ -1,13 +1,10 @@
-NETWORK=
+NETWORK=localhost
 
 PRIVATE_KEY='0x123'
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)